mozilla-release/browser/components/BrowserGlue.jsm

```diff
@@ -3093,14 +3093,43 @@ BrowserGlue.prototype = {
     XPIProvider._cliqz_UpdateCliqzExtToLatest();
   },
 
+  _cliqz_handleBrowserUpdate() {
+    // Custom Cliqz updatecheck function to handle functions to
+    // be executed on update or install
+    // RECURRING: make sure this is changed at every release
+    const CLIQZ_UI_VERSION = 1;
+    const CLIQZ_MIGRATION_PREF = "cliqz.browser.migration.version";
+
+    // Put all functions here which you want to execute at every startup
+    PermissionsUtils.importFromPrefs("blockautoplay.", "autoplay-media");
+
+    if (!Services.prefs.prefHasUserValue(CLIQZ_MIGRATION_PREF)) {
+      // This is a new profile, nothing to migrate.
```

Comment:
This mean that users, who will receive this version after 1.34 (without 1.35 update) will not get our fixes at all

Reply:
Updated the coe